cookiewow-wordpress.php

```diff
@@ -75,6 +75,29 @@ function cookiewow_sanitize_settings_fields( $fields ) {
 function cookiewow_settings_section_description() {
 }
 
+function cookiewow_uninstall() {
+	delete_option( 'cookiewow_option' );
+}
+
+function cookiewow_wp_head() {
+	$option = get_option( 'cookiewow_option' );
+
+	if ( ! isset( $option['cookiewow_token'] )
+		|| '' === trim( $option['cookiewow_token'] ) ) {
```

Comment:
This is the comparison style of WordPress. The value will be at the front of comparison clause.

For an example, https://github.com/WordPress/WordPress/blob/dc3a76e8f325e05eca7534babf9e42cdc14bb5bd/wp-admin/includes/taxonomy.php#L132.